### PR TITLE
HELPS-2529-  Update "User Aptitude - JWT - Email Claim" test package.json

### PR DESCRIPTION
**Pull request by MaggieMaSS**
When running the "User Aptitude - JWT - Email Claim" test, the test initiates, but then freezes and never completes.
It turns out that we need to add the `optionalClaims` and add the email address to it.
The permission for user in `package.json` should be like this: 
```

      "user": {
        "optionalClaims": [
          {          
            "value": "email"
          }
        ]
      },
```

### CODE REVIEW

File: ldk/javascript/examples/self-test-loop/package.json [hunk 1]
```diff
@@ -44,7 +44,14 @@
...
       "process": {},
       "ui": {},
-      "user": {},
+      "user": {
+        "optionalClaims":[
+          {
+            "value" :"add your email address here"
+          }
+        ]
+
```
Comment (JosefBud-olive) [inline ldk/javascript/examples/self-test-loop/package.json:53]: Extra line break here

File: ldk/javascript/examples/self-test-loop/package.json [hunk 2]
```diff
@@ -44,7 +44,14 @@
       "filesystem": {},
       "process": {},
       "ui": {},
-      "user": {},
+      "user": {
+        "optionalClaims":[
+          {
+            "value" :"add your email address here"
```
Comment (JosefBud-olive) [inline ldk/javascript/examples/self-test-loop/package.json:50]: Should this include an actual email address for the self test loop?

**Review by JosefBud-olive — COMMENTED** [review]
Could you add a description? 😁

File: ldk/javascript/examples/self-test-loop/package.json (hunk 2 above)
Comment (MaggieMaSS) [reply ldk/javascript/examples/self-test-loop/package.json:50]: I used my email on my end 0.0. Do you volunteer to put your email here? Or I can just put Brett's 0,0

Comment (JosefBud-olive) [reply ldk/javascript/examples/self-test-loop/package.json:50]: @MaggieMaSS if we use a Google Group email for our team, that should do the trick!

File: ldk/javascript/examples/self-test-loop/src/config/user-group.ts [hunk 3]
```diff
@@ -21,6 +21,6 @@ export const userTestGroup = (): TestGroup =>
       'User Aptitude - JWT - Email Claim',
       userTests.testJwtIncludeEmail,
       10000,
-      'No action required',
+      'Please check package.json and update the value of email to your email address!',
```
Comment (JosefBud-olive) [inline ldk/javascript/examples/self-test-loop/src/config/user-group.ts:24]: This is outdated now that we added an e-mail address

File: ldk/javascript/examples/self-test-loop/package.json [hunk 4]
```diff
@@ -44,7 +44,13 @@
       "filesystem": {},
       "process": {},
       "ui": {},
-      "user": {},
+      "user": {
+        "optionalClaims":[
+          {
+            "value" :"team-ldx@oliveai.com"
```
Comment (MichaelOliveMcCarrick) [inline ldk/javascript/examples/self-test-loop/package.json:50]: Shouldn't this be 'email'? Does this actually work?

**Review by MichaelOliveMcCarrick — APPROVED** [review]

**Review by JosefBud-olive — APPROVED** [review]
Look like the space is in the wrong place (`value :email` instead of `value: email`) but not going to hold up the PR over it

**Pull request merged**